backend/server.py

```diff
@@ -40,7 +43,31 @@ def upload_file():
         
         samples = GoogleSensorParser([f.filename]).parse_files()
 
-        return samples
+        return {'type': 'upload', 'data': samples}
+
+@app.route('/stats', methods = ['POST'])
+def compute_stats():
```

Comment:
Document your functions

Reply:
Python doc strings should be after the function def, before the code block

def compute_stats():
  """Handles requests for stats data and responds with the requested
      stats.
  Attributes:
      request.data: The data sent from the frontend.
          request.data.channels: Key value pairs of channels and arrays
          to compute statistics for.
  """